src/OgDeleteOrphansBase.php

```diff
@@ -135,10 +146,17 @@ protected function deleteOrphan($entity_type, $entity_id) {
       return;
     }
 
-    // Only delete content that is fully orphaned, i.e. it is no longer
+    // Only delete group content that is fully orphaned, i.e. it is no longer
     // associated with any groups.
-    $group_count = $this->membershipManager->getGroupCount($entity);
-    if ($group_count == 0) {
+    if ($this->groupAudienceHelper->hasGroupAudienceField($entity->getEntityTypeId(), $entity->bundle())) {
+      $group_count = $this->membershipManager->getGroupCount($entity);
```

Comment:
Only do a group count if the entity is actually group content.

Reply:
```suggestion
      // Only do a group count if the entity is actually group content.
      $group_count = $this->membershipManager->getGroupCount($entity);
```